### ISSUE TITLE
Automatically select block

### ISSUE THREAD

**Issue by johnstonphilip** [youknowriad/blockbook#9]
Since there is only 1 block being shown in the BlickList at a time, it would save a click if the block was automatically selected, making the popover toolbar appear and all of those things.

**Comment by youknowriad** [comment]
Makes sense. I think we need to find a nice place to render the inspector panel too.
